src/components/footer/footer.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Button from '@material-ui/core/Button';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/container';
+import Grid from '@material-ui/core/grid';
+
+import IconLink from './iconlink';
+import facebook from './facebook.svg';
+import github from './github.svg';
+import medium from './medium.svg';
+import twitter from './twitter.svg';
+
+const styles = theme => ({
+	container: {
+		backgroundColor: theme.palette.primary.dark
+	},
+	buttonLink: {
+		textDecoration: 'none'
+	},
+	content: {
+		padding: theme.spacing(1, 3),
+		alignItems: 'center',
+		fontFamily: ['Poppins', 'sans-serif'],
+		fontSize: theme.typography.fontSize,
+		color: '#FFFFFF',
+		textAlign: 'center'
+	},
+	icons: {
+		display: 'flex',
+		justifyContent: 'center',
+		alignItems: 'center',
+		paddingTop: theme.spacing(1)
+	},
+	footerText: {
+		fontSize: theme.typography.fontSize * 0.875
+	},
+	button: {
+		fontSize: theme.typography.fontSize * 0.875
+	}
+});
+
+class Footer extends React.Component {
+	render() {
+		const { classes } = this.props;
+		return (
+			<div className={classes.container}>
+				<Container maxWidth="md">
```

Comment:
Since we are using the `Container`, we can merge `classes.container` div with `Container`. You can inject the CSS property of the `Container` with the `classes` props. Read the docs for the `Container` API.